### ISSUE TITLE
Don't include blobs in chain info query responses

### ISSUE THREAD

**Issue by ndr-ds** [linera-io/linera-protocol#2301]
https://github.com/linera-io/linera-protocol/pull/2220#discussion_r1689387313

**Comment by ndr-ds** [comment]
We need to make it so that you can't have a ChainInfoQuery return more than 1 of the following 3 at the same time:

* `ChainManager`'s `proposed` -> should be fine to return the whole `BlockProposal` after we add the limits on #2199 
* `ChainManager`'s `proposed_blobs` -> Same as above
* `ChainManager`'s `locked_blobs` -> This one might have published _and_ read blobs, so we need to implement a custom way of returning the locked blobs: maybe sending the blobs one by one and the caller can aggregate them. This could technically also be solved by streaming the blobs as the response.

**Comment by afck** [comment]
Yes, I think the `ChainInfoQuery` should have a field like `request_locked_blob: Option<BlobId>` or maybe `request_locked_blobs: Vec<BlobId>`.

**Comment by ndr-ds** [comment]
We could technically do `request_locked_blobs` and calculate how many we can request at once based on the current blob size limit and gRPC message size limit 🤔 so we minimize the amount of chain info queries we do
